packages/components/src/color-palette/index.js

```diff
@@ -55,42 +55,42 @@ export default function ColorPalette( { colors, disableCustomColors = false, val
 				);
 			} ) }
 
-			{ ! disableCustomColors &&
-				<Dropdown
-					className="components-color-palette__item-wrapper components-color-palette__custom-color"
-					contentClassName="components-color-palette__picker"
-					renderToggle={ ( { isOpen, onToggle } ) => (
-						<Tooltip text={ customColorPickerLabel }>
-							<button
+			<div className="components-color-palette__custom-clear-wrapper">
+				{ ! disableCustomColors &&
+					<Dropdown
+						className="components-color-palette__custom-color"
+						contentClassName="components-color-palette__picker"
+						renderToggle={ ( { isOpen, onToggle } ) => (
+							<Button
 								type="button"
```

Comment:
`type` `button` is applied inside `Button` component. It should be removed here.

See: https://github.com/WordPress/gutenberg/blob/master/packages/components/src/button/index.js#L42

Reply:
Good call! Removed that in 62087ba. Should be all set. 👍